doc/api/process.markdown

```diff
@@ -648,6 +648,7 @@ Will print something like:
       zlib: '1.2.8',
       ares: '1.10.0-DEV',
       modules: '43',
+      icu: '55.1', // only if built with i18n support
```

Comment:
Not sure that comment is necessary. Strictly speaking, it'd also have to be there for `openssl`.